backend/etcd/etcd.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	// StateDir is the base path for Sensu's local storage.
-	StateDir = "/var/lib/sensu"
+	StateDir = "/var/lib/sensu/sensu-backend"
```

Comment:
I believe we'll need to update `backend/cmd/start.go`

Reply:
There is little chance that this will affect anyone and will not conclude in any data loss.

The only scenario in which this will affect anyone is if a user is not using a config file (or has removed the `state-dir` line from the backend config) and has manually created `/var/lib/sensu` themselves and applied the appropriate ownership/permissions changes.

Our current behaviour is causing issues for users as packaging works as intended, but the backend binary does not.

Reply:
The worst case scenario is that a user will need to either:
1. Specify the state dir when starting the backend with the `-d` flag.
2. Add `state-dir` to their existing backend.yml.
3. Move the contents of `/var/lib/sensu` into `/var/lib/sensu/sensu-backend`.

Reply:
A user adding a configuration file doesn't necessarily mean they also configured the state dir, does it?